Dockerfile-default-rocm

```diff
@@ -1,40 +1,138 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
+#RUN sudo sysctl -w kernel.core_pattern="core-%h-%e-%s-%t-%u-%p"
```

Comment:
guessing some of these commented-out lines were for debugging?

Reply:
removed.